### ISSUE TITLE
Table: Accessibility issues with reading Table row/header using JAWS screen reader software

### ISSUE THREAD

**Issue by manishcharsap** [SAP/ui5-webcomponents#8833]
### Describe the bug

When accessing the row in the table, the contents/ information’s present within the rows and columns in the “Keys” table is not announced

### Reproduction steps

1. Open JAWS screen reader software.
2. Go to the table and access table row/header using tab. 
3. 
...

### Expected Behaviour

All the contents / information’s should be conveyed to the user. Here in the table the information/ contents present in the rows and column should be announced to the user when the rows and columns are accessed, and the correct row and the column and row count information of the table should be read out correctly to the screen reader user when accessed.

### Screenshots or Videos

<img width="924" alt="PTBS-38912-tabel-issue" src="https://github.com/SAP/ui5-webcomponents-react/assets/103047215/4678ce9c-80c4-4b10-90c7-0ec9d3c1d61b">

### UI5 Web Components for React Version

1.19.0

### UI5 Web Components Version

1.17.0

### Browser

Chrome

### Operating System

windows

### Organization

SAP Labs India

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by NHristov-sap** [comment]
Hi @manishcharsap ,

Please provide working isolated sample that demonstrates the issue.

Best Regards,
Nikolay Hristov
UI5 Web Components GitHub Dispatcher

**Comment by manishcharsap** [comment]
<img width="962" alt="PTBS-38912-tabel-issue-app" src="https://github.com/SAP/ui5-webcomponents/assets/103047215/75ac6c20-4bfd-496f-ba2a-f3962a831b79">

codesandbox link:
https://t4h73v.csb.app/

**Comment by NHristov-sap** [comment]
Hi @manishcharsap ,

Have you tried to switch the Virtual Cursor of the JAWS ( Ins + Z )? Then you get the whole information instead of reading only the focused object.

Best Regards,
Nikolay Hristov
UI5 WebComponents Dispatcher

**Comment by manishcharsap** [comment]
When switch the Virtual Cursor of the JAWS ( Ins + Z ) then its working fine.

Thanks @NHristov-sap
